Fatal panic in fork choice

```
panic: bitlists are different lengths

goroutine 291 [running]:
github.com/prysmaticlabs/go-bitfield.Bitlist.Overlaps(0xc023922440, 0x12, 0x12, 0xc029986220, 0x12, 0x12, 0x0)
        external/com_github_prysmaticlabs_go_bitfield/bitlist.go:148 +0x1cd
github.com/prysmaticlabs/prysm/beacon-chain/operations/attestations.(*Service).seen(0xc00fd48480, 0xc0111083c0, 0xf, 0x126, 0xc011e94080)
        beacon-chain/operations/attestations/prepare_forkchoice.go:110 +0x12f
github.com/prysmaticlabs/prysm/beacon-chain/operations/attestations.(*Service).batchForkChoiceAtts(0xc00fd48480, 0x1a59540, 0xc0000440e8, 0x0, 0x0)
        beacon-chain/operations/attestations/prepare_forkchoice.go:52 +0x392
github.com/prysmaticlabs/prysm/beacon-chain/operations/attestations.(*Service).prepareForkChoiceAtts(0xc00fd48480)
        beacon-chain/operations/attestations/prepare_forkchoice.go:28 +0x133
created by github.com/prysmaticlabs/prysm/beacon-chain/operations/attestations.(*Service).Start
        beacon-chain/operations/attestations/service.go:48 +0x3f

```

--- a/beacon-chain/operations/attestations/prepare_forkchoice.go
+++ b/beacon-chain/operations/attestations/prepare_forkchoice.go
@@ -107,7 +107,7 @@ func (s *Service) seen(att *ethpb.Attestation) (bool, error) {
 		if !ok {
 			return false, errors.New("not a bit field")
 		}
-		if savedBitlist.Overlaps(att.AggregationBits) {
+		if savedBitlist.Len() == att.AggregationBits.Len() && savedBitlist.Overlaps(att.AggregationBits) {
 			return true, nil
 		}
 	}

